portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,39 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function sayHello() {
```

Comment:
Can this function be deleted? I think you referencing `showMessages()` below in your html?